package.js

```diff
@@ -46,6 +46,8 @@ Package.onUse(function (api) {
 
   // Non-core packages
   api.use("aldeed:template-extension@3.4.3");
+  // Should be replaced with straight up built-in 'typescript' in Meteor 1.8.2
+  api.use("adornis:typescript@0.9.14");
```

Comment:
Note that this is a temporary package someone built to maintain the abandoned `barbatus:typescript`, which is in turn deprecated by first-class TypeScript support in Meteor 1.8.2.

So it's not clear if we'd need this at all, depending on the Meteor version we're using.